src/app/input/input.component.scss

```diff
@@ -0,0 +1,120 @@
+@import '../../styles/variables.scss';
+@import '../../styles/mixins.scss';
+
+.input-group {
+  width: 100%;
+  margin-top: 30px;
+  position: relative;
+  height: auto;
+  padding-bottom: 30px;
+  margin-bottom: 20px;
+  .input-icon {
+    height:$fs-18;
+    position:absolute;
+    right:0;
+    top:10px;
+  }
+  label {
+    color: $gray-medium;
+    font-size: $fs-18;
+    font-weight: normal;
+    position: absolute;
+    pointer-events: none;
+    left: 0px;
+    top: 10px;
+    transition: 0.2s ease all;
+    -moz-transition: 0.2s ease all;
+    -webkit-transition: 0.2s ease all;
+  }
+  label.theme-dark {
+    color: $gray-dark;
+  }
+  .input-field {
+    font-size: $fs-18;
+    padding: 10px 10px 10px 5px;
+    display: block;
+    width: 100%;
+    border: none;
+    color: white;
+    border-bottom: 1px solid $gray-medium;
+    background: transparent;
+    padding-bottom:20px;
+    font-weight:$fw-light;
+    &:focus {
+      outline: none;
+    }
+  }
+  .input-field.theme-dark{
```

Comment:
You can write this as 
`.input-field{
     &.theme-dark{
     }
}`
 also @live-wire